db/migrate/20150902221944_update_users.rb

```diff
@@ -0,0 +1,6 @@
+class UpdateUsers < ActiveRecord::Migration
+  def change
+    add_column :users, :first_name, :string
+    add_column :users, :last_name, :string
+  end
+end
```

Comment:
When I sign up with GitHub, field "name" is filled with my Name and Surname I have on GitHub profile (as one string). When I go to profile page I have to fill this again (now as two strings). Why do I have to do this? And why there is now 3 fields in the table with the same information? Is there a way to fetch the name while user is signing up with GitHub and show it at her profile page?